src/main/java/mate/controller/AssigningDriverToCar.java

```diff
@@ -0,0 +1,55 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AssigningDriverToCar extends HttpServlet {
+    private DriverService driverService;
+    private CarService carService;
+    private List<Car> cars;
+    private List<Driver> drivers;
```

Comment:
don't need to make these variables global, make them local

Reply:
I use this fields in two methods.